docs/blog/2019-10-17-how-gatsby-can-power-your-site-offline-even-without-javascript/index.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "How Gatsby can power your site offline - even without JavaScript"
+date: 2019-10-17
+author: David Bailey
+excerpt: >
```

Comment:
```suggestion
excerpt: "With Gatsby's offline plugin, network troubles are a thing of the past."
```
Should this be included as a direct string child? I wasn't sure what the angled bracket was for.

Reply:
I've seen other articles using the angled bracket for longer excerpts - doesn't seem necessary here since the excerpt is quite short, so I agree with you.